tests/unit_tests/test_camera_efficiency.py

```diff
@@ -33,12 +33,29 @@ def telescope_model(set_db):
     return telescopeModel
 
 
+@pytest.fixture
```

Comment:
Duplication from definition in test_telescope_model. Should we avoid duplications and move this into conftest.py?

Reply:
Yeah, that's a good idea, I will add these to conftest.py